src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return calculator;
+    },
+
+    subtract(num) {
+      this.result -= num;
+
+      return calculator;
+    },
+
+    multiply(num) {
+      this.result *= num;
+
+      return calculator;
+    },
+
+    divide(num) {
+      this.result /= num;
+
+      return calculator;
+    },
```

Comment:
don't need to return calculator object in these methods
please check again an example

<img width="856" alt="Screenshot 2023-10-17 at 14 00 14" src="https://github.com/mate-academy/js_advanced_calculator/assets/64746943/a1c009ff-68e3-4f56-bbb8-c56c00b99630">

```suggestion
    add(num) {
      this.result += num;
    },

    subtract(num) {
      this.result -= num;
    },

    multiply(num) {
      this.result *= num;
    },

    divide(num) {
      this.result /= num;
    },
```